src/components/DataRoom/IndustryComparison/utils/updateCanvasDimensions.ts

```diff
@@ -0,0 +1,20 @@
+export const updateCanvasDimensions = (
+  canvasRef: React.RefObject<HTMLCanvasElement>,
+  containerRef: React.RefObject<HTMLDivElement>,
```

Comment:
Instead of passing the ref, I think we can use the element directly, e.g.
```suggestion
  canvas: HTMLCanvasElement,
  container: HTMLDivElement,
```